plugins/droid/droid-codec.cpp

```diff
@@ -568,7 +602,19 @@ bool DroidVideoDecoder::createCodec()
         droid_media_codec_set_callbacks(m_codec, &cb, this);
     }
 
-    {
+    if (!m_disable_media_buffers) {
+      m_buffer_queue = droid_media_codec_get_buffer_queue (m_codec);
```

Comment:
Very minor, but the indentation isn't quite right here.

Reply:
Fixed, thank you.